### PR TITLE
020 clean code lodash

### PR DESCRIPTION
**Pull request by aknipey**
Fixed f'up from 17, added isUndefined where applicable. Deleted old code comments

### CODE REVIEW

File: frontend/src/components/FinalBarPlot.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Data, Layout, Shape } from "plotly.js";
 import { normalise, percentageScale } from "./utils/utils";
 import { makeExtendedSimulation } from "./OutcomeGraph";
 import { Outcome, Simulation } from "./customTypings/simulation";
+import _ from "lodash";
```
Comment (patrickemoore) [inline frontend/src/components/FinalBarPlot.tsx:10]:
```suggestion
import { isUndefined } from "lodash";
```

File: frontend/src/components/customTypings/simulation.tsx [hunk 2]
```diff
@@ -47,3 +47,15 @@ export type PortfolioState = {
 export type Outcome = PortfolioState[];
 
 export type Simulation = Outcome[];
+
```
Comment (patrickemoore) [inline frontend/src/components/customTypings/simulation.tsx:50]: type

**Review by patrickemoore — CHANGES_REQUESTED** [review]
fix import

File: frontend/src/components/customTypings/simulation.tsx (hunk 2 above)
Comment (aknipey) [reply frontend/src/components/customTypings/simulation.tsx:50]: for what?

**Review by patrickemoore — APPROVED** [review]

**Pull request merged**